<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Windows10/Downloads/YouTube_23392.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Windows10/Downloads/soundcloud_icon-icons.com_66677.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>51</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Windows10/Downloads/youtube_music_social_network_song_multimedia_icon_192250.png</pixmap>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>131</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_yt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
    </property>
    <property name="text">
     <string>Просмотры на YouTube</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(58, 58, 58);
	border-radius: 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_ytmsuic">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>49</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="text">
      <string>Просмотры на YouTube Music</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_soundcloud">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="text">
      <string> Прослушивания на SoundCloud</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>125</y>
       <width>431</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	border-radius: 10px;
	background-color: rgb(50, 50, 50);
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 107, 150, 255), stop:1 rgba(130, 255, 174, 255));
}</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_views_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="text">
      <string>Загрузка..</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_views_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="text">
      <string>Загрузка...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_views_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	border-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Загрузка...</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(58, 58, 58);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/Windows10/Downloads/button.ico</normaloff>C:/Users/Windows10/Downloads/button.ico</iconset>
     </property>
     <property name="autoRepeatInterval">
      <number>100</number>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>label_yt</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
